roles/ckan/templates/kubernetes/ckan_volumes.yaml

```diff
@@ -61,7 +61,7 @@ spec:
   storageClassName: "{{ storage_class_name }}"
 {% endif %} 
   accessModes:
-    - ReadWriteMany
+    - ReadWriteOnce
```

Comment:
So this one probably needs to be a ternary if its RWO in AWS, as Azure needs RWM?

Reply:
Agreed. Want me to do it in this branch or will you do that in yours?

Reply:
Yours pls 😄